08-projects.Rmd

```diff
@@ -9637,38 +9691,41 @@ Make sure to replace "\<title id here\>" with the `title_id` of your favorite sh
 - The first 5 results from your query.
 ```
 
-##### 7. Write a query that adds the rating to the end of each episode. To do so, use the query you wrote in (6) as a subquery. Was this also your favorite episode?
+##### 7. Write a query that adds the rating to the end of each episode. To do so, use the query you wrote in (6) as a subquery. ich episode has the highest rating? Is it also your favorite episode?
 
-**Relevant topics:** [inner join](#sql-inner-join), [aliasing](#sql-aliasing), subqueries, [desc](#sql-examples), [limit](#sql-examples), [order by](#sql-order-by)
+**Relevant topics:** [INNER JOIN](#sql-inner-join), [aliasing](#sql-aliasing), subqueries, [ORDER BY](#sql-order-by), DESC, LIMIT
+
+**Note**: Various helpful examples that utilize the relevant topics in this problem can be found [here](#sql-examples).
 
 ```{block, type="bbox"}
 **Item(s) to submit:**
 
 - SQL query used to solve the problem in a code chunk.
 - The `episode_title_id`, `primary_title`, and `rating` of the top rated episode from the tv series from (5).
-- A statement saying whether it is also your favorite episode.
+- A statement saying whether the highest rated episode is also your favorite episode.
 ```
 
-##### 8. Write a query that returns the `season_number` (from the `episodes` table), and average `rating` (from the `ratings` table) for each season. Write another query that only returns the season number and `rating` for the highest rated season. Consider the highest rated season the season with the highest average.
+##### 8. Write a query that returns the `season_number` (from the `episodes` table), and average `rating` (from the `ratings` table) for each season of your favorite TV show from (5). Write another query that only returns the season number and `rating` for the highest rated season. Consider the highest rated season the season with the highest average.
 
-**Relevant topics:** [inner join](#sql-inner-join), [aliasing](#sql-aliasing), [group by](#sql-groupby), having, [avg](#sql-avg)
+**Relevant topics:** [INNER JOIN](#sql-inner-join), [aliasing](#sql-aliasing), [GROUP BY](#sql-groupby), [AVG](#sql-avg)
 
 ```{block, type="bbox"}
 **Item(s) to submit:**
 
-- The 2 SQL queries used to solve the problems in a code chunk.
+- The 2 SQL queries used to solve the problems in two code chunks.
```

Comment:
It seems that when having more than one query in an SQL code chunk, not all the output would be included in the knitted file.

Reply:
Thanks!